### PR TITLE
Home dev

### PR DESCRIPTION
**Pull request by betoobernal**
## Summary

* Add and Design home page
* Refactor css files
* Create Header Component
* Create Card Component

## Screenshots

image

## Trello Card

https://trello.com/c/fy11suPP/17-agregar-un-home-al-sitio

### CODE REVIEW

File: src/app/screens/Home/components/Card/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import { Link } from 'react-router-dom';
+
+import styles from './styles.module.scss';
+
+function Card({ data }) {
+  return (
+    <div className={styles.card}>
+      <i className={`material-icons ${styles.cardIcon}`}>{data.icon}</i>
+      <div>
+        <div className={styles.cardInfo}>{data.info}</div>
```
Comment (FrankIglesias) [inline src/app/screens/Home/components/Card/index.js]: Span

File: src/app/screens/Home/components/Card/index.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import styles from './styles.module.scss';
+
+function Card({ data }) {
+  return (
+    <div className={styles.card}>
+      <i className={`material-icons ${styles.cardIcon}`}>{data.icon}</i>
+      <div>
```
Comment (FrankIglesias) [inline src/app/screens/Home/components/Card/index.js]: Extra div?

File: src/app/screens/Home/components/Card/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,32 @@
+@import '../../../../../scss/variables/colors';
```
Comment (FrankIglesias) [inline src/app/screens/Home/components/Card/styles.module.scss]: Remove the ../

File: src/app/screens/Home/index.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+import Card from './components/Card';
+import styles from './styles.module.scss';
+
+import ROUTES from 'constants/routes';
+
+function Home() {
+  const teamsCards = [
```
Comment (FrankIglesias) [inline src/app/screens/Home/index.js]: Move the constants above the function

File: src/app/screens/Home/index.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+      info: 'Create a new repo',
+      descriptionAction: 'create',
+      path: ROUTES.REPO_CREATION,
+      icon: 'book'
+    }
+  ];
+
+  return (
+    <div className={styles.container}>
+      <div className={`page-header ${styles.pageHeader}`} data-parallax="true">
```
Comment (FrankIglesias) [inline src/app/screens/Home/index.js]: Span

File: src/app/screens/Home/index.js [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+        Wolox Repo Manager
+      </div>
+      <div className={styles.section}>
+        <h2 className={styles.sectionTitle}>Teams</h2>
+        <div className={styles.sectionBody}>
+          {teamsCards.map(card => (
```
Comment (FrankIglesias) [inline src/app/screens/Home/index.js]: Remove parenthesis

File: src/app/screens/Home/index.js [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+            <Card key={card.path} data={card} />
+          ))}
+        </div>
+      </div>
+      <div className={styles.section}>
+        <h2 className={styles.sectionTitle}>Repositories</h2>
+        <div className={styles.sectionBody}>
+          {repositoriesCards.map(card => (
```
Comment (FrankIglesias) [inline src/app/screens/Home/index.js]: Same

File: src/app/screens/Home/index.js (hunk 6 above)
Comment (FrankIglesias) [reply src/app/screens/Home/index.js]: Add key value

File: src/app/screens/Home/styles.module.scss [hunk 8]
```diff
@@ -0,0 +1,37 @@
+@import '../../../scss/variables/colors';
```
Comment (FrankIglesias) [inline src/app/screens/Home/styles.module.scss]: Same

File: src/app/components/Routes/index.js [hunk 9]
```diff
@@ -8,6 +8,7 @@ import Routes from '../../../constants/routes';
 
 import AuthenticatedRoute from './components/AuthenticatedRoute';
 
+const Home = lazy(() => import('../../screens/Home'));
```
Comment (FrankIglesias) [inline src/app/components/Routes/index.js]: Use alias for all imports

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: src/app/screens/Home/index.js (hunk 6 above)
Comment (betoobernal) [reply src/app/screens/Home/index.js]: * The ESLint considers that would be like that. (Include parenthesis).
* Already exist key value.

File: src/app/screens/AddMember/index.js [hunk 10]
```diff
@@ -17,19 +15,16 @@ class AddMemberContainer extends Component {
...
-            <div className="container">
-              <div className="row">
-                <div className="col-md-6 ml-auto mr-auto">
-                  <AddMember
-                    onSubmit={this.handleSubmit}
-                    memberAdded={this.props.memberAdded}
-                    loading={this.props.loading}
-                  />
-                </div>
+          <div className="container">
```
Comment (FrankIglesias) [inline src/app/screens/AddMember/index.js]: check extra divs  here

File: src/app/screens/AddMember/layout.js [hunk 11]
```diff
@@ -2,20 +2,18 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Field, reduxForm } from 'redux-form';
 
-import SimpleSpinner from '../../components/SimpleSpinner';
-
-import styles from './styles.module.scss';
-
 import InputLabelNew from '~components/InputLabelNew';
 
+import SimpleSpinner from '../../components/SimpleSpinner';
```
Comment (FrankIglesias) [inline src/app/screens/AddMember/layout.js]: use alias

File: src/app/screens/AddOwner/layout.js [hunk 12]
```diff
@@ -3,20 +3,18 @@ import PropTypes from 'prop-types';
 import { t } from 'i18next';
 import { Field, reduxForm } from 'redux-form';
 
-import SimpleSpinner from '../../components/SimpleSpinner';
-
-import styles from './styles.module.scss';
-
 import InputLabelNew from '~components/InputLabelNew';
 
+import SimpleSpinner from '../../components/SimpleSpinner';
```
Comment (FrankIglesias) [inline src/app/screens/AddOwner/layout.js]: alias

File: src/app/screens/Home/index.js [hunk 13]
```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+import ROUTES from '~constants/routes';
+
+import Card from './components/Card';
+import styles from './styles.module.scss';
+
+const teamsCards = [
```
Comment (FrankIglesias) [inline src/app/screens/Home/index.js]: use i18n later

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: src/app/components/Header/styles.module.scss [hunk 14]
```diff
@@ -0,0 +1,4 @@
+.page-header {
+  background-image: url('/assets/img/BG3.jpg');
```
Comment (FrankIglesias) [inline src/app/components/Header/styles.module.scss]: change the name of the file

File: src/app/screens/AddMemberToTeam/index.js [hunk 15]
```diff
@@ -31,21 +29,16 @@ class AddTeamToMemberContainer extends Component {
...
-                    loading={this.props.loading}
-                  />
-                </div>
-              </div>
+          <div className="row">
+            <div className="col-10 col-md-6 col-xl-4 m-auto">
+              <AddTeamToMember
+                onSubmit={this.handleSubmit}
+                memberAdded={this.props.memberAdded}
+                data={this.state.data.map(team => ({ label: team.name, value: team }))}
```
Comment (FrankIglesias) [inline src/app/screens/AddMemberToTeam/index.js]: move this to a constant above the return line

File: src/app/screens/Home/styles.module.scss [hunk 16]
```diff
@@ -0,0 +1,37 @@
+@import './src/scss/variables/colors';
+
+.container {
+  background: $alabaster;
+  height: 100vh;
+}
+
+.page-header {
+  background-image: url('/assets/img/BG3.jpg');
```
Comment (FrankIglesias) [inline src/app/screens/Home/styles.module.scss]: change the name

File: src/app/screens/Login/layout.js [hunk 17]
```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import { t } from 'i18next';
 
-import { useAuth0 } from '../../../react-auth0-spa';
+import { useAuth0 } from 'react-auth0-spa';
```
Comment (FrankIglesias) [inline src/app/screens/Login/layout.js]: remove the empty line above

File: src/app/screens/Teams/index.js [hunk 18]
```diff
@@ -20,18 +19,14 @@ class TeamsContainer extends Component {
...
-                <div className="col-md-6 ml-auto mr-auto">
-                  <TeamCreation
-                    onSubmit={this.handleSubmit}
-                    isError={this.props.isError}
-                    loading={this.props.loading}
-                    teamCreated={this.props.teamCreated}
-                  />
-                </div>
-              </div>
+          <div className="row">
```
Comment (FrankIglesias) [inline src/app/screens/Teams/index.js]: extra div nesting?

File: src/index.js [hunk 19]
```diff
@@ -2,13 +2,16 @@ import React from 'react';
...
 
-import { Auth0Provider } from './react-auth0-spa';
-import App from './app';
-import config from './auth_config.json';
-import './config/i18n';
+import history from '~utils/history';
+
 import './scss/application.scss';
+
+import '~config/i18n';
```
Comment (FrankIglesias) [inline src/index.js]: move this to line 6

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**